packages/node_modules/@webex/webex-core/src/lib/services/index.js

```diff
@@ -14,8 +15,10 @@ registerInternalPlugin('services', Services, {
   }
 });
 
+export {constants};
 export {default as ServiceInterceptor} from './interceptors/service';
 export {default as ServerErrorInterceptor} from './interceptors/server-error';
 export {default as Services} from './services';
 export {default as ServiceCatalog} from './service-catalog';
+export {default as ServiceHost} from './service-host';
```

Comment:
Exporting the new `ServiceHost` class here, as it needs to be handed off downstream.